app/views/user/orders/index.html.erb

```diff
@@ -1,6 +1,11 @@
-<h1>My Orders</h1>
+<h1><%= "#{current_user.name}'s Orders" %></h1>
 
-<% @user_orders.each do |order| %>
-  <h3><%= link_to "Order #{order.id}" %></h3>
-  <p><%= order.status.titleize %></p>
+<% current_user.orders.each do |order| %>
+  <div id="order-<%= order.id %>">
+    <h3><%= link_to "Order #{order.id}", user_order_path(order) %></h3>
+    <p>Placed on: <%= order.created_at %></p>
+    <p>Last Updated: <%= order.updated_at %></p>
+    <p>Status: <%= order.status %></p>
```

Comment:
Side note: cart_check_out_spec line 36 will need to be lower-case "pending"

Reply:
Done!